oauth2/heroku_test.go

```diff
@@ -1,4 +1,4 @@
-package oauth2_test
+package oauth2
```

Comment:
Good catch. Will fix! We renamed it because we were testing private functionality of the oauth package.